pkg/utils/bcd.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package utils
+
+import (
+	"github.com/yerden/go-util/bcd"
+)
+
+var standard = &bcd.BCD{
+	Map: map[byte]byte{
```

Comment:
We have BTI inline with the library but we don't do it with EBCIDIC. Should these but part of this library or packages?

Reply:
The part is used to specify blank field of BCD encoding/decoding.
so default is 0x0 (the value is not the value that used in the bcd library)